documentation/src/docs/asciidoc/release-notes/release-notes-5.12.0-M1.adoc

```diff
@@ -86,6 +86,11 @@ JUnit repository on GitHub.
   a test-scoped `ExtensionContext` in `Extension` methods called during test class
   instantiation. This behavior will become the default in future versions of JUnit.
 * `@TempDir` is now supported on test class constructors.
+* `@ParameterizedTest`s now support argument count validation as an experimental feature.
```

Comment:
I haven't found any other release not referencing whether it's experimental, but given the nature of experimental features potentially being removed or changed, it seemed like a sensible thing to include.